exercises/flatten-array/canonical-data.json

```
@@ -168,6 +168,38 @@
         2
       ]
     },
+    {
+      "uuid": "c6cf26de-8ccd-4410-84bd-b9efd88fd2bc",
+      "description": "consecutive null values at the front of the list are omitted from the final result",
+      "property": "flatten",
+      "input": {
+        "array": [
+          null,
+          null,
+          3
+        ]
+      },
+      "expected": [
+        3
+      ]
+    },
+    {
+      "uuid": "382c5242-587e-4577-b8ce-a5fb51e385a1",
+      "description": "consecutive null values at the middle of the list are omitted from the final result",
```

```suggestion
      "description": "consecutive null values in the middle of the list are omitted from the final result",
```

while I think "in the middle of" seems more natural than "at the middle of", since English isn't my first language, it's very possible I am wrong.

---

I speak English as a first language, and I think either works.  But looking at it again, I agree that `in` is a bit more natural.  😄

---

There is another one, if we wanted to use `in`.

---

Hmm... I am a bit confused. 
(I am not native English speaker neither.)

So, the final decision would be
> in the middle of the list 
and 
> at the front of the list
?

I will make sure I avoid these grammar mistakes in the future. Thanks :D